### ISSUE TITLE
Faces missing when importing

### ISSUE THREAD

**Issue by ArcadiaLabs** [Siccity/GLTFUtility#58]
Hi,
I made a model in Sketchup and exported it to gltf using the Khronos plugin. When I import the gltf in Unity, some faces are missing, while everything looks good in the Windows 3D viewer (and sketchup, obviously).

I'm not sure if I'm doing something wrong or if there is a bug in the importer.

**Edit** : _seems there is something wrong with the Khronos gltf exporter itself : I opened the gltf directly in Unity without using your GLTFUtility, and I have the same issue. I also redesigned the missing face without success. Now I don't understand why it looks good in the windows 3d viewer..._

Here is the gltf file : 
[house_2.zip](https://github.com/Siccity/GLTFUtility/files/4217097/house_2.zip)
The missing face is the top floor upper face. It should look like this (screenshot from windows 3d viewer) : 
![house_2](https://user-images.githubusercontent.com/6577107/74699476-117bea80-5201-11ea-964d-dbb2f178f3ef.png)
But in unity, the top face is missing :
![house_2_unity1](https://user-images.githubusercontent.com/6577107/74699803-f198f680-5201-11ea-9f69-dad6c461a4af.png)
while the bottom face looks ok : 
![house_2_unity2](https://user-images.githubusercontent.com/6577107/74699838-08d7e400-5202-11ea-93dc-16e67823490c.png)

**Comment by Siccity** [comment]
Does the console log say anything when importing?

**Comment by ArcadiaLabs** [comment]
No, there is no error in the console when importing. I also imported the gltf to online gltf viewers, and they look ok.

**Comment by luizcarlosfx** [comment]
I think your importer may not be importing correctly the double sided option, from what I see your doubleSided property on GltfMaterial is never used

**Comment by Siccity** [comment]
Nice find! Yes that could very well be it. I will have a look at that asap

**Comment by subhradeepc** [comment]
Any updates on this ? Getting the same issue as well

**Comment by Siccity** [comment]
> Any updates on this ? Getting the same issue as well

Can you check if maybe the face is there when you look at it from its backside?

**Comment by Holi0317** [comment]
Sees that #118 will fix this issue. Didn't go through the whole PR but seems the change did mention `doubleSided` and `culling`.

In the meantime I am adding solidify modifier to my blender model before export to gltf as a workaround described here: https://github.com/KhronosGroup/glTF-Blender-IO/issues/154#issuecomment-455747814
